static/js/containers/VideoDetailPage.js

```diff
@@ -222,6 +199,82 @@ class VideoDetailPage extends React.Component<*, void> {
         </div>
       </DocumentTitle>
     )
+  } 
+
+  toggleAnalyticsOverlay () {
+    this.props.dispatch(actions.videoUi.toggleAnalyticsOverlay())
+  }
+
+  renderVideoPlayer = (video: Video) => {
+    const { videoUi } = this.props
+    if (videoIsProcessing(video)) {
+      return (
+        <div className="video-message">
+          Video is processing, check back later
+        </div>
+      )
+    }
+
+    if (videoHasError(video)) {
+      return <div className="video-message">Something went wrong :(</div>
+    }
+
+    return (
+      <div
+        className="video-player-container"
+        style={{
+          position: "relative"
+        }}
+      >
+        <VideoPlayer
+          videoPlayerRef={ref => {
+            this.videoPlayerRef = ref
+          }}
+          video={video}
+          cornerFunc={this.updateCorner}
+          selectedCorner={videoUi.corner}
+          analyticsOverlayIsVisible={videoUi.analyticsOverlayIsVisible}
+        />
+        {videoUi.analyticsOverlayIsVisible
+          ? this.renderAnalyticsOverlay()
+          : null}
+      </div>
+    )
+  }
+
+  renderAnalyticsOverlay() {
```

Comment:
Codecov indicates that this function isn't covered, could you add a unit test for it? And if possible/necessary, additional unit tests [to reach 95% coverage](https://codecov.io/gh/mitodl/odl-video-service/pull/563/diff).

Reply:
PS There are also conflicts with the master branch that need to be resolved.